### PR TITLE
Create Freestyle Project by Iris2024

### PR DESCRIPTION
**Pull request by Iris20221**
[FreeStyleProjectByIrisTest created](https://trello.com/c/zK3n7WbJ/59-tc0000101-dashboard-new-item-create-freestyle-project)

### CODE REVIEW

**Review by antonilyu — APPROVED** [review]

File: src/test/java/school/redrover/FreeStyleProjectByIrisTest.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        wait.until(visibilityOfElementLocated(By.xpath("//*[@id='tasks']/div[1]/span/a"))).click();
+        wait.until(visibilityOfElementLocated(By.className("jenkins-input"))).sendKeys("new Freestyle project");
+        wait.until(visibilityOfElementLocated(By.xpath("//button[@type = 'submit']"))).click();
+        wait.until(visibilityOfElementLocated(By.xpath("//span[contains(text(),'Freestyle project')]"))).click();
+        wait.until(visibilityOfElementLocated(By.xpath("//button[@type= 'submit']"))).click();
+        wait.until(visibilityOfElementLocated(By.xpath("//h1[contains(text(),'Configure')]")));
+        wait.until(visibilityOfElementLocated(By.xpath("//button[contains(text(),'Save')]"))).click();
+        WebElement textWebElement = wait.until(visibilityOfElementLocated(By.xpath("//h1[contains(text(),'new Freestyle project')]")));
+
+        assertEquals(textWebElement.getText(), "new Freestyle project");
```
Comment (IrynaKolh) [inline src/test/java/school/redrover/FreeStyleProjectByIrisTest.java:34]: Не подскажите как правильнее использовать Assert.assertEquals или assertEquals?

File: src/test/java/school/redrover/FreeStyleProjectByIrisTest.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import static org.openqa.selenium.support.ui.ExpectedConditions.visibilityOfElementLocated;
+import static org.testng.Assert.assertEquals;
+import static org.testng.AssertJUnit.assertTrue;
+
+public class FreeStyleProjectByIrisTest extends BaseTest {
+
+    @Test
+    public void testCreateFreestyleProject() {
+
+        WebDriverWait wait = new WebDriverWait(getDriver(), Duration.ofSeconds(5));
```
Comment (IrynaKolh) [inline src/test/java/school/redrover/FreeStyleProjectByIrisTest.java:20]: Просто интересно, вейтеры здесь необходимы тк элемент не успевает появиться или это просто на всяк случай?

**Review by IrynaKolh — APPROVED** [review]
LGTM

File: src/test/java/school/redrover/FreeStyleProjectByIrisTest.java (hunk 2 above)
Comment (Iris20221) [reply src/test/java/school/redrover/FreeStyleProjectByIrisTest.java:20]: It allows us to wait for a certain condition to be true before proceeding with the execution of the script. This is particularly useful in cases where elements on the page might take some time to load or become interactable.

File: src/test/java/school/redrover/FreeStyleProjectByIrisTest.java (hunk 1 above)
Comment (Iris20221) [reply src/test/java/school/redrover/FreeStyleProjectByIrisTest.java:34]: It's the same stuff, but it is pretty common to use static imports for assertions (Assert class methods, and not only) - this improves code readability.
Please also note that for some cases, static import must not be used. E.g.: `List.of(<list elements>)`, if you see `of(<list e;ements>)`, it won't make any sense.

**Pull request merged**